packages/react-server-cli/src/startServer.js

```diff
@@ -128,3 +143,33 @@ const handleCompilationErrors = (err, stats) => {
 		// behavior based on whether or not --minify is set.
 	}
 }
+
+const setupLogging = (logLevel) => {
+	logging.setLevel('main',  logLevel);
+	// TODO: the time and gauge log levels should also be parameters.
+	if (process.env.NODE_ENV !== "production") { //eslint-disable-line no-process-env
+		logging.setLevel('time',  'fast');
+		logging.setLevel('gauge', 'ok');
+	}
+}
```

Comment:
I like that this happens when I call `startServer` directly now.  I'd like it _more_ if it happened in the _browser_, too! ;)